src/main/java/com/algorand/algosdk/transaction/Transaction.java

```diff
@@ -175,367 +175,6 @@ public class Transaction implements Serializable {
     @JsonProperty("spmsg")
```

Comment:
@ahangsu Minor:  Visual reminder to add a CHANGELOG once we get closer to the finish line.